### ISSUE TITLE
Custom Dataset

### ISSUE THREAD

**Issue by EscVM** [iwyoo/DIRNet-tensorflow#11]
Hi @iwyoo,

thank you for this repository!

Is there a way to train the model with a custom set of images?
If yes, how can I do that?

Thanks.

**Comment by EscVM** [comment]
I think I managed to do that! But, is it correct that the loss function increases? Is it a similarity measure between the two input images?

**Comment by studentguoyiyuan** [comment]
> I think I managed to do that! But, is it correct that the loss function increases? Is it a similarity measure between the two input images?

Hi~
Did you solve this problem by change your images to the form of MNIST?

**Comment by takerujason** [comment]
Dear @iwyoo @EscVM 
Could you guide me how to import own‘s datasets (.png).
Thanks!
